fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/MetricMaker.java

```diff
@@ -187,7 +170,7 @@ protected TemplateDruidQuery getMergedQuery(List<String> names) {
      */
     protected PostAggregation getNumericField(String fieldName) {
         // Get the field
-        MetricField field = getDependentQuery(fieldName).getMetricField(fieldName);
+        MetricField field = metrics.get(fieldName).getMetricField();
```

Comment:
The previous method call did:
foo = metrics.get(fieldName)
bar = return foo.getTemplateDruidQuery
bar.getMetricField(fieldName)

How is this not the same?